bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,374 @@
+            "name": data["name"],
+            "url": data["url"],
+            "description": f"{data['summary']}\n\n" if "summary" in data else "\n",
+            "release_date": release_date,
+            "rating": round(data["total_rating"] if "total_rating" in data else 0, 2),
+            "rating_count": data["total_rating_count"] if "total_rating_count" in data else "?",
+            "platforms": ", ".join(platform["name"] for platform in data["platforms"]) if "platforms" in data else "?",
+            "status": GameStatus(data["status"]).name if "status" in data else "?",
+            "age_ratings": rating,
+            "made_by": companies,
+            "storyline": data["storyline"] if "storyline" in data else ""
+        }
+        page = GAME_PAGE.substitute(formatting)
+
+        return page, url
+
+    async def search_games(self, search_term: str) -> List[str]:
+        """Search game from IGDB API by string, return listing of pages."""
+        lines = []
+
+        # Define request body of IGDB API request and do request
+        body = SEARCH_BODY.substitute({"term": search_term})
+
+        async with self.http_session.get(url=f"{BASE_URL}/games", data=body, headers=HEADERS) as resp:
+            data = await resp.json()
+
+        # Loop over games, format them to good format, make line and append this to total lines
+        for game in data:
+            formatting = {
+                "name": game["name"],
+                "url": game["url"],
+                "rating": round(game["total_rating"] if "total_rating" in game else 0, 2),
+                "rating_count": game["total_rating_count"] if "total_rating" in game else "?"
+            }
+            line = GAME_SEARCH_LINE.substitute(formatting)
+            lines.append(line)
+
+        return lines
+
+    async def get_companies_list(self, limit: int, offset: int = 0) -> List[Dict[str, Any]]:
+        """Get random Game Companies from IGDB API."""
+        # Create request body from template
+        body = COMPANIES_LIST_BODY.substitute({
+            "limit": limit,
+            "offset": offset
+        })
+
+        async with self.http_session.get(url=f"{BASE_URL}/companies", data=body, headers=HEADERS) as resp:
+            return await resp.json()
+
+    async def create_company_page(self, data: Dict[str, Any]) -> Tuple[str, str]:
+        """Create good formatted Game Company page."""
+        # Generate URL of company logo
+        url = LOGO_URL.substitute({"image_id": data["logo"]["image_id"] if "logo" in data else ""})
+
+        # Try to get found date of company
+        founded = dt.utcfromtimestamp(data["start_date"]).date() if "start_date" in data else "?"
+
+        # Generate list of games, that company have developed or published
+        developed = ", ".join(game["name"] for game in data["developed"]) if "developed" in data else "?"
+        published = ", ".join(game["name"] for game in data["published"]) if "published" in data else "?"
```

Comment:
this is a great example of a readable, well-written comprehension and ternary expression.